spec/classes/backend_resources_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe 'sensu::backend::resources', :type => :class do
+  on_supported_os({facterversion: '3.8.0'}).each do |os, facts|
+    context "on #{os}" do
+      let(:facts) { facts }
+      context 'with default values for all parameters' do
+        it { should compile }
+        it { should have_sensu_namespace_resource_count(1) }
```

Comment:
I left the count tests so that there is validation on how many defaults we're adding to avoid rogue additions.  Updated with more complete unit tests.